spec/spec_wbparsing.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'spec_helper'
+require_relative "../parsing_scripts/account"
+require_relative "../parsing_scripts/wb_parser"
+
+describe WBParser do 
+
+  let(:nokogiri) {Nokogiri::HTML}
+  let(:get_user_name) {nokogiri.fragment(File.read("fixtures/username.html"))}
+  let(:get_account_balance) {nokogiri.fragment(File.read("fixtures/account_description.html"))}
+  let(:get_account_currency) {nokogiri.fragment(File.read("fixtures/account_description.html"))}
+  
+  describe "#get_user_name" do
+    it "get account username" do
+      expect(subject.send(:get_user_name, get_user_name)).to eq "Anastasia Monosienco"
```

Comment:
Please update your code to parse all data for 1 account and show that in test. like `account.should == {account_hsh}`  
Same for transactions.